tests/standard_workarounds_test.py

```diff
@@ -94,6 +94,11 @@ def get_table_title_from_id(standard, table_id):
 #     assert 'Macro Attributes' not in table_title, 'Table title now contains suffix "Macro Attributes"'
 
 
+# def test_missing_word_in_table_a_89_4_1_title(part03):
```

Comment:
If this is no longer needed, let's just delete it instead of commenting it out.

Reply:
Ah, sorry I didn't realize this. If this is what the README says and it's what we've been doing, then I'd leave it commented out.